evalml/tests/automl_tests/test_automl_search_classification.py

```diff
@@ -1637,3 +1637,48 @@ def test_automl_passes_allow_long_running_models_iterative(
         estimators.extend(["CatBoost Classifier", "XGBoost Classifier"])
 
     assert "Dropping estimators {}".format(", ".join(sorted(estimators))) in caplog.text
+
+
+@patch(
+    "evalml.automl.automl_algorithm.default_algorithm.DefaultAlgorithm._create_ensemble",
+    return_value=[],
+)
+@patch("evalml.pipelines.components.estimators.Estimator.fit")
+@patch(
+    "evalml.pipelines.MulticlassClassificationPipeline.score",
+    return_value={"Log Loss Multiclass": 0.5},
+)
+@pytest.mark.parametrize("allow_long_running_models", [True, False])
+@pytest.mark.parametrize("unique", [10, 200])
+def test_automl_passes_allow_long_running_models_default(
```

Comment:
Why not adjust the existing test to parametrize over both algos?

Reply:
Good call, fixing that now!